### DISCUSSION TITLE
What is "Default Delay Hours"?

### DISCUSSION THREAD

**Discussion by bradsk88** [TandoorRecipes/recipes#2285] [Q&A] [answered]
Hello, I'm new to Tandoor and trying to get the hang of it.
I noticed this on the shopping list but I don't understand what it's supposed to do.

![image](https://user-images.githubusercontent.com/2147630/215240519-3b172771-073b-4675-8515-af9ff082b8f1.png)

Can anyone explain this to me? Thanks

**Comment by smilerz** [comment, marked as answer]
In the shopping list you have the option to “delay” an item by 24 hours (default). This will hide those items for that length of time.  For example if you are at the butcher and want to hide all of the produce.  

You can change the 24 hour delay to some other period.

**Reply by bradsk88** [reply]
Ahh. Gotcha. 

So it's kind of like a "snooze" feature.
